ledger-service/http-json/src/it/scala/http/AuthorizationTest.scala

```diff
@@ -74,19 +80,18 @@ final class AuthorizationTest
 
   behavior of "PackageService against an authenticated sandbox"
 
-  // TEST_EVIDENCE: Authorization: Updating the package service fails with insufficient authorization
-  it should "fail immediately if the authorization is insufficient" in withLedger {
+  it should "fail immediately if the authorization is insufficient" taggedAs authorizationSecurity in withLedger {
```

Comment:
This seems to drop contents from the `TEST_EVIDENCE` comment, namely:
```
Authorization: Updating the package service fails with insufficient authorization
```

Feel free to rewrite / enhance information, but do not drop information.